### PR TITLE
Update README.md to match own project

### LINKED ISSUE #26: Update README.md to match own project

### PR DESCRIPTION
**Pull request by Feng1231**
Closes #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,22 @@
...
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
+  * It is developed by a team of 5 from NUS Computer Science as a project for a module on software engineering (CS2103T). You can find more [about us here](https://ay2425s1-cs2103t-f14a-1.github.io/tp/AboutUs.html).
+
+* For the detailed documentation of this project, see the **[UG Teach Product Website](https://ay2425s1-cs2103t-f14a-1.github.io/tp/)**.
+
+
+# Acknowledgement
 * This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
```
Comment (Soumyadip-cmd) [inline README.md:25]: Acknowledge the original source of the code e.g.,
This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).

File: README.md [hunk 2]
```diff
@@ -4,13 +4,22 @@
...
+**UGTeach** empowers undergraduate private tutors to efficiently manage payments, track student performance, and organize schedules. It streamlines tutoring operations, ensuring you stay organized and focused on helping your students achieve their academic goals.<br>
+
+  **Example usages:**
+  * As a private tutor, you can add student's personal information and their tuition details
+  * As a private tutor, you can view the list of students you are currently tutoring
+  * As a private tutor, you can view the amount of money you have earned from tutoring
+
+# More Information
+  * It is **written in OOP fashion**. 
+  * It is **written in Java**, and has about 6 kLoC.
```
Comment (Soumyadip-cmd) [inline README.md:17]: 6K LoC instead of 6 kLoC

**Review by Soumyadip-cmd — COMMENTED** [review]
I think need to put in the acknowldegement. Other than a minor error, the rest looks good

File: README.md (hunk 2 above)
Comment (Feng1231) [reply README.md]: I have changed `6 kLoC` to `6 KLoC`. The template given to us initially was 6 KLoC. Would this be ok? @Soumyadip-cmd

Comment (Soumyadip-cmd) [reply README.md]: Ok then LGTM

**Review by Soumyadip-cmd — APPROVED** [review]
Looks good.

**Pull request merged**